### ISSUE TITLE
mod-card: a border is displayed around a card in 2022.11

### ISSUE THREAD

**Issue by ildar170975** [thomasloven/lovelace-card-mod#230]
My Home Assistant version: 2022.11.1

My lovelace configuration method (GUI or yaml): storage / yaml

I have a lot of decluttering templates for a card or a row (supposed to be used inside Entities card).
Some of them have `mod-card` to use card-mod, like:
```
some_decluttering_template:
  card:
    type: custom:mod-card
    card:
      type: custom:multiple-entity-row
      ...
    card_mod:
      style:
        ...
```
or
```
some_decluttering_template:
  card:
    type: custom:mod-card
    card:
      type: custom:hui-element
      ...
    card_mod:
      style:
        ...
```
Since 2022.11.0 these cards got a border.

First, I suspected a `decluttering-card`.
But then I concluded that `mod-card` gets this border.

Check this test example:
```
decluttering_templates:
  decl_test_hui_element:
    default:
      - NAME: xxx
    card: &ref_card
      type: custom:hui-element
      row_type: sensor-entity
      entity: sun.sun
      name: '[[NAME]]'

  decl_test_hui_element_card_mod:
    default:
      - NAME: xxx
    card:
      type: custom:mod-card
      card: *ref_card
      card_mod:
        style:
          hui-element $ hui-generic-entity-row $: |
            .text-content {
              color: red;
            }
```
The test card:
```
type: entities
entities:
  - type: custom:decluttering-card
    template: decl_test_hui_element
    variables:
      - NAME: decl_test_hui_element
  - type: custom:decluttering-card
    template: decl_test_hui_element_card_mod
    variables:
      - NAME: decl_test_hui_element_card_mod
```
The 2nd row gets a border:
![изображение](https://user-images.githubusercontent.com/71872483/200148489-9d4e6825-927f-46d2-8dff-b5d60464ad4b.png)

Then I tested `mod-card` w/o using a `decluttering-card`:
```
type: entities
entities:
  - type: custom:hui-element
    row_type: sensor-entity
    entity: sun.sun
    name: hui-element
  - type: custom:mod-card
    card:
      type: custom:hui-element
      row_type: sensor-entity
      entity: sun.sun
      name: hui-element
```
The 2nd row with `mod-card` gets a border:
![изображение](https://user-images.githubusercontent.com/71872483/200148540-e7b2666f-5a39-46fc-b906-e69feae538fa.png)

Probably a reason of this is that the `mod-card` inherits a border of a parent card.
Here is a fix for this:
![изображение](https://user-images.githubusercontent.com/71872483/200148601-8f595f6e-2359-4ba2-838f-b416e44fe60d.png)

Error messages from the browser console:
none

---

**By putting an X in the boxes ([]) below, I indicate that I:**

- [x] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [x] Have made sure I am using the latest version of the plugin.

- [x] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [x] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by ildar170975** [comment]
Another example - using `mod-card` for `vertical-stack`:
```
type: custom:mod-card
style:
  hui-vertical-stack-card$: |
    .card-header {
      font-size: 10px !important;
      color: red !important;
    }
card:
  type: vertical-stack
  title: Changed font for title
  cards:
    - type: entities
      entities:
        - entity: sun.sun
        - entity: sun.sun
    - type: entity
      entity: sun.sun
```
The stack gets a border:
![изображение](https://user-images.githubusercontent.com/71872483/200148723-5e55d137-ba3c-49f1-b1a8-c0c264183ebc.png)

**Comment by ildar170975** [comment]
I believe this could be solved by https://github.com/thomasloven/lovelace-card-mod/pull/231

**Comment by EuleMitKeule** [comment]
I have the same problem. Do you have a quick fix for this while your PR isn't merged yet by any chance?
I tried putting `:host { border-style: none !important; } on pretty much every place inside my theme file without any effect.

**Comment by Francois1976** [comment]
same here.  Thanks

**Comment by EuleMitKeule** [comment]
Putting this `ha-card-border-width: 0` into your theme file fixes the issue.

**Comment by ildar170975** [comment]
> into your theme file fixes the issue.

This is NOT a fix. This is a workaround for SOME cases.
And some people DO like borders.
This "fix" causes removing all borders:
![image](https://user-images.githubusercontent.com/71872483/202702392-99ff1c9d-6eeb-49ab-83f7-7b6a299acc9a.png)

**Comment by EuleMitKeule** [comment]
True, for some cases it might be enough though.

**Comment by thomasloven** [comment]
Fixed by #231

Thanks!
